electronics/README.asciidoc

```diff
@@ -0,0 +1,69 @@
+# GibiBot Electronics Documentation
+
+## Information
+
+The mainboard is controlled through a STM32G441KBT6 microcontroller. It uses the USB connector to communicate with computer. The microcontroller is powered by the USB connector through a 5V to 3.3V voltage regulator(NCP164CSN330T1G). It can be programmed through the programmer connector with a STLINK programmer.
+
+The board has two isolated sides which have both have their own seperate ground planes. 2 MAX14930DASE+ isolators and 1 MAX12930EASA+ isolator connect both sides. The microcontroller sends signals through the isolators to the motor drivers, servos and the thrower.
+
+The microcontroller side has a debug LED which can be controlled through the microcontroller.
+
+The driver side has 3 PDRV8243HRXYQ1 motor drivers which can power 3 motors. The motor drivers need an nsleep signal from the microcontroller to wake up. The drivers are controllerd with a direction signal and a PWM signal. The direction pin determines which way the motor spins.

```

Comment:
> can power 3 motors

Specify which type of motors.

controllerd -> controlled

Reply:
Fixed.